<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>descriptiveComboBox</class>
 <widget class="QWidget" name="descriptiveComboBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>79</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>DescriptiveComboBox</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_name">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_description">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <italic>true</italic>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>descriptiveComboBox</receiver>
   <slot>on_element_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>element_changed(object)</signal>
  <slot>on_element_changed(int)</slot>
 </slots>
</ui>
